### DISCUSSION TITLE
Change User name

### DISCUSSION THREAD

**Discussion by sakhavhyand** [innightwolfsleep/llm_telegram_bot#135] [Q&A]
Hi, maybe I missed something but is it possible to change the user name for the bot ? 
Maybe using the telegram user name ? I'm currently looking at the code to find something like this but with no success.

**Comment by innightwolfsleep** [comment]
You can use double minus prefix. Send message "--MyNewName" to switch your name during conversation.
Also, you can add "user" string in character json file.

**Reply by sakhavhyand** [reply]
I tried the double minus before but the bot just responded. Looking closely I've just noted that telegram had transformed the double minus into a special symbol. It indeed works well. Sorry for that.

But what do you think of the idea to use the telegram name ? Could it be possible ?

**Reply by innightwolfsleep** [reply]
I think this can be added as option in future.

**Reply by sakhavhyand** [reply]
Okay, thanks for the answers.
